src/b-selectable.html

```diff
@@ -94,8 +96,15 @@
                 },
 
                 activate: function() {
```

Comment:
I understand why you did this change, but I think we could do better by listening to key events only when the selectable is focused, and removing the listener when it is unfocused.